apps/monitors/monocker.md

```diff
@@ -0,0 +1,52 @@
+# Monocker
+
+Monocker monitors Docker (MONitors dOCKER) containers and alerts on 'state' change.
+There is no web ui or fancy dashboard.
+
+<br>
+
+- [Github repo](https://github.com/petersem/monocker)
+
+
+## docker-compose
+
+```yml
+---
+services:
+  monocker:
+    container_name: monocker
+    image: petersem/monocker
+    environment:
+      SERVER_LABEL: 'Dev'
+      MESSAGE_PLATFORM: 'telegram@your_bot_id@your_chat_id'
+      # MESSAGE_PLATFORM: 'pushbullet@your_api_key@your_device_id'
+      # MESSAGE_PLATFORM: 'pushover@your_user_key@your_app_api_token'
+      # MESSAGE_PLATFORM: 'discord@webhook_url'
+      LABEL_ENABLE: 'false'
+      ONLY_OFFLINE_STATES: 'false''
+      EXCLUDE_EXITED: 'false'. 
```

Comment:
the dot at the end probably shouldn't be there?

Reply:
Deleted the extra apostrophe and the dot